devicetypes/smartthings/zigbee-non-holdable-button.src/zigbee-non-holdable-button.groovy

```diff
@@ -0,0 +1,216 @@
+/*
+ *  Copyright 2016 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy
+ *  of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+    definition(name: "Zigbee Non-Holdable Button", namespace: "smartthings", author: "SmartThings", runLocally: false, mnmn: "SmartThings", vid: "generic-button-4", ocfDeviceType: "x.com.st.d.remotecontroller") {
```

Comment:
@dkirker `generic-button-4` includes the "Holdable Button" capability. I wonder, does it make any sense for that vid to include that capability or can we remove it? If we do need to keep it there, I think we should create new UI Metadata without that capability and use it here. Thoughts?

Reply:
Hmmmm I'm not sure that `Holdable Button` is a capability that is active anymore. It is marked deprecated. I don't *think* there would be a negative impact with removing it from the metadata. However, I'm not entirely certain. The `Button` capability basically eclipses `Holdable Button`.